course-definition.yml

```
@@ -445,7 +445,10 @@ stages:
 
       **Note:** You only need to focus on one backreference and one capturing group in this stage. We'll get to handling multiple backreferences in the next stage.
     marketing_md: |
-      In this stage, you'll add support for single backreferences. You'll implement support for `\d`.
+      In this stage, you'll add support for single backreferences. You'll implement support for `\N`.
```

```suggestion
      In this stage, you'll add support for single backreferences. You'll implement support for `\1`.
```

Since we're only looking at single backreferences here, `\1` would be more clear